### ISSUE TITLE
Modify Docker section in Readme

### ISSUE THREAD

**Issue by regbo** [keylase/nvidia-patch#588]
Hello,

I should have known this from the start, but the readme on this repo is unclear when it comes to docker. I read the docker section as a requirement, not as an alternative, if this patch can't be installed on the host. I would suggest changing:

> It is possible to use this patch with nvidia-docker containers, even if host machine hasn't patched drivers. See Dockerfile for example.

to

> If you are unable to install this patch on a host machine, it is possible to use this patch within nvidia-docker containers. See Dockerfile for example.

**Comment by vadimstasiev** [comment]
I completely agree.  It was only after I stumbled upon this that I realized I had to install the patch solely on the host for it to work inside docker. Can the readme be updated to literally say this?

**Comment by vadimstasiev** [comment]
I completely agree. It was only after I stumbled upon this that I realized I had to install the patch solely on the host for it to work inside docker. Can the readme be updated to literally say this?
